salt/apps/ocw/sync_repo.sls

```diff
@@ -0,0 +1,54 @@
+add_private_github_ssh_key:
+  file.managed:
+    - name: /root/.ssh/ocw_ssh_key
+    - contents_pillar: {{ salt.pillar.get('ocw:github_ssh_key') }}
```

Comment:
Remove the `salt.pillar.get` it should just be the string `ocw:github_ssh_key` when you're using contents_pillar, because the file.managed state will do the work of pulling the value out of that key path in the assigned pillar data.